### PR TITLE
add a zh doc introducing the new features of kubekey

### PR DESCRIPTION
**Pull request by 24sama**
Signed-off-by: 24sama <leo@kubesphere.io>

### What type of PR is this?

/kind documentation

### What this PR does / why we need it:
Add a zh doc introducing the new features of kubekey.
Please take a look at this PR, thanks.
/cc @kubesphere/sig-installation @kubesphere/sig-docs 

### Which issue(s) this PR fixes:

Fixes #

### Special notes for reviewers:
```
```

### Does this PR introduced a user-facing change?

```release-note

```

### Additional documentation, usage docs, etc.:

```docs

```

### CODE REVIEW

File: docs/manifest-example.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  - arch: amd64
+    type: linux
+    id: centos
+    version: "7"
+    osImage: CentOS Linux 7 (Core)
+    repository:
+      iso:
+        localPath:
+        url: https://github.com/pixiake/k8s-dependencies/releases/download/v1.0.0/centos-7-amd64-rpms.iso
+  kubernetesDistributions: # Define the kubernetes distribution that will be included in the artifact.
```
Comment (FeynmanZhou) [inline docs/manifest-example.md:29]: How many k8s distributions does KK support?

File: docs/zh/manifest_and_artifact.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+KubeKey v2.0.0 版本（以下简称 kk ）新增了清单（manifest）和制品（artifact）的概念，为用户离线部署 Kubernetes 集群提供了一种解决方案。在过去，用户需要准备部署工具，镜像 tar 包和其他相关的二进制文件，每位用户需要部署的 Kubernetes 版本和需要部署的镜像都是不同的。现在使用 kk 的话，用户只需使用清单`manifest`文件来定义将要离线部署的集群环境需要的内容，再通过该`manifest`来导出制品`artifact`文件即可完成准备工作。离线部署时只需要 kk 和 `artifact`就可快速、简单的在环境中部署镜像仓库和 Kubernetes 集群。
+
+## 什么是 KubeKey 清单 （manifest）？
+`manifest`就是一个描述当前 Kubernetes 集群信息和定义`artifact`制品中需要包含哪些内容的文本文件。目前有两种方式来生成该文件：
+* 根据模版手动创建并编写该文件
+* 使用 kk 命令根据已存在的集群生成该文件
+
+第一种方式需要了解该配置文件中不同字段的更多信息，请参考 [manifest-example.md](../manifest-example.md)。
+
+以下内容仅针对于第二种由 kk 实现的生成文件的方式。
```
Comment (FeynmanZhou) [inline docs/zh/manifest_and_artifact.md:11]: 方法二是对已有集群的配置信息进行扫描然后生产一个新的 manifest？

Comment (FeynmanZhou) [reply docs/zh/manifest_and_artifact.md:11]: 对已有集群没有要求？

Comment (24sama) [reply docs/zh/manifest_and_artifact.md:11]: 没有要求，kk 根据默认路径或者指定的kubeconfig连接上集群，然后就执行扫描集群信息的操作了。

**Pull request merged**